### PR TITLE
Update the Swedish translation

### PR DESCRIPTION
**Pull request by karllindmark**
I'm updating the Swedish translation 1) for the new English strings, as well as 2) rephrasing strings to use more suitable terms/formatting (in terms of capitalization and so on).

However, I'm not completely convinced that the current "förvärvare" is the best term for "assignee". I'd rather use "ansvarig", which unfortunately is more towards "maintainer" and "responsible" in the English language. Any thoughts on that?

Also, let's ping @stillesjo for some help reviewing the language changes. :-)

### CODE REVIEW

**Comment by hzsweers** [comment]
@Meisolsson can you review?

**Comment by Meisolsson** [comment]
Looking at it right now
On Aug 5, 2015 09:25, "Henri (Zac) Sweers" <notifications@github.com> wrote:@Meisolsson can you review?

**Comment by Meisolsson** [comment]
Just a thought, why not use "gren" instead of "branch"?

File: app/src/main/res/values-sv/strings.xml [hunk 1]
```diff
@@ -186,52 +195,52 @@
...
     <string name="password">Lösenord</string>
-    <string name="username_or_email">Användarnamn eller Email</string>
-    <string name="login_or_email">Användarnamn eller Email</string>
+    <string name="username_or_email">Användarnamn eller epost</string>
+    <string name="login_or_email">Användarnamn eller epost</string>
     <string name="followers">Prenumeranter</string>
     <string name="following">Prenumererar på</string>
     <string name="follow">Följ</string>
-    <string name="unfollow">Sluta följa</string>
+    <string name="unfollow">Avfölja</string>
```
Comment (Meisolsson) [inline app/src/main/res/values-sv/strings.xml]: Should this not be "Avfölj"?

File: app/src/main/res/values-sv/strings.xml [hunk 2]
```diff
@@ -243,17 +252,18 @@
...
     <string name="disable_wrapping">Inaktivera wrapping</string>
     <string name="code">Kod</string>
-    <string name="following_user">Följer personer…</string>
-    <string name="unfollowing_user">Slutar följa personer…</string>
-    <string name="starring_repository">Stjärnmärk…</string>
-    <string name="unstarring_repository">Ta bort stjärnmärkning…</string>
-    <string name="deleting_repository">Raderar repository…</string>
-    <string name="navigate_to">Navigera till…</string>
+    <string name="following_user">Följer personer&#8230;</string>
+    <string name="unfollowing_user">Avföljer personer&#8230;</string>
```
Comment (Meisolsson) [inline app/src/main/res/values-sv/strings.xml]: Change to "användare" instead of "personer"

File: app/src/main/res/values-sv/strings.xml [hunk 3]
```diff
@@ -243,17 +252,18 @@
...
+    <string name="starring_repository">Stjärnmärk&#8230;</string>
```
Comment (Meisolsson) [inline app/src/main/res/values-sv/strings.xml]: "Stjärnmärker"?

File: app/src/main/res/values-sv/strings.xml [hunk 4]
```diff
@@ -243,17 +252,18 @@
...
+    <string name="unstarring_repository">Ta bort stjärnmärkning&#8230;</string>
```
Comment (Meisolsson) [inline app/src/main/res/values-sv/strings.xml]: "Tar bort stjärnmärkning..."

File: app/src/main/res/values-sv/strings.xml [hunk 5]
```diff
@@ -120,56 +124,61 @@
...
-    <string name="edit_milestone">Ändra Milstolpe</string>
+    <string name="edit_milestone">Ändra milstolpe</string>
     <string name="edit_assignee">Ändra Förvärvaren</string>
     <string name="description">Beskrivning</string>
     <string name="gist_description_hint">Gist skapad i Android</string>
     <string name="title">Titel</string>
     <string name="edit">Ändra</string>
     <string name="star">Stjärnmärk</string>
-    <string name="starring_gist">Stärnmärker Gist…</string>
+    <string name="starring_gist">Stärnmärker gist&#8230;</string>
```
Comment (Meisolsson) [inline app/src/main/res/values-sv/strings.xml]: Typo? ("Stjärnmärker")

**Comment by Meisolsson** [comment]
Think that's it, about "assignee" I like "ansvarig" since it's an easier word. But it does have a slightly different meaning

File: app/src/main/res/values-sv/strings.xml (hunk 1 above)
Comment (karllindmark) [inline app/src/main/res/values-sv/strings.xml]: Correct, will be addressed in my next commit!

**Comment by stillesjo** [comment]
I agree that "ansvarig" is the way to go since it has a better connection towards assignee than "förvärvare". "Förvärva" is more about gathering or acquiring things than it is about being assigned to a task.

**Comment by karllindmark** [comment]
@Meisolsson: I vote that we keep "branch" :-)

**Comment by stillesjo** [comment]
@Meisolsson 
Isn't branch as commonly used by swedes as repository/repo (instead of gren)?

**Comment by karllindmark** [comment]
@Meisolsson: I've rebased my branch, so it should be ready for merge if we're content with the changes.

**Comment by Meisolsson** [comment]
@stillesjo Since I don't usually talk to very many swedes about software development I actually don't know. But I guess it would be.

**Comment by Meisolsson** [comment]
Looks good to me

**Pull request merged**